shared/cfg/base.cfg

```diff
@@ -265,19 +265,19 @@ image_unbootable_pattern = "Hard Disk.*not a bootable disk"
 
 #
 # List of hypervisor-monitor object names (one per guest),
-#    used to communicate with hypervisor to control guests.
-#    Order cooresponds to 'vms' list above.
-monitors = hmp1
+# used to communicate with hypervisor to control guests.
+# Order cooresponds to 'vms' list above.
+monitors = qmpmonitor1
 # hmp1 monitor type (protocol), if only hmp1 type is going to be set
-#monitor_type_hmp1 = human
+# monitor_type_hmp1 = human
 # Default monitor type (protocol), if multiple types to be used
-monitor_type = human
+monitor_type = qmp
 # If set catch_monitor, will start another monitor in qemu for
 # VmRegister and ScreenDump threads.
 catch_monitor = catch_monitor
 #monitor_type_catch_monitor = qmp
 # Pattern to get vcpu threads from monitor.both support
-vcpu_thread_pattern = "thread_id.?[:|=]\s*(\d+)"
+vcpu_thread_pattern = u'thread_id':\s+(\d+)
```

Comment:
It's not correct, suggest to keep using the original one.

Reply:
@luckyh 

Thanks,there is a typo of " ' ".